quickstarts/quickstart-template-guide.md

```diff
@@ -0,0 +1,141 @@
+# Quickstart Guide
+
+:information_source: Read this document before you start working on the [quickstart template](template-quickstarts.md)
+
+## Introduction
+
+A quickstart introduces your users to your application for the first time.
+It focuses on the **primary feature** of the application and helps your users to start using the application as quickly as possible.
+
+A good quickstart answers the following questions:
+
+- Scope:
+ - What is the core purpose of this application?
+ - What is the minimal use case that is covered in this quickstart?
+- Install (if applicable):
+ - How do I download, install, and configure the application?
+ - How do I install and configure the application?
+ - How can I get any required access keys or authentication credentials?
+- Hello  World:
+ - How can I run a simple workflow for a core feature or a common use? case.
+- Next steps:
+ - What other features are available to explore in the application?
+
+A Quickstart guide is often confused with a Getting Started  guide. Though both these documents help the users get acquainted with the application they widely differ in their target audience. 
+
+There are instances where a Marketing guide is often confused with a Quickstart and a Getting Started guide. The inline table gives a brief comparison about the differences. 
+
+| |Quickstart Guide|Getting Started Guide| Marketing Guide|
+|--------|----------------|----------------------|---------------|
+ |Target Audience|Domain Experts: Know the problem space and are aware of what they need to do|Beginners:New to problem space and the product|Business Decision Maker: Makes strategic decisions on whether to buy the product 
+|Content|Minimal conceptual information|Detailed conceptual information on product/domain| No/less conceptual information. Instead focuses on the benefits and the customers using the product.
+|Focus|how-to| how-to and why|  sales
+
+## Why do I need a quickstart?
+
+A quickstart is often the first opportunity for your user to build an opinion about the technical quality of your project. It can:
+- Make users comfortable using the application.
+- Work with any application version the user might use.
+- Lead to the willingness to handle complex workflows.
+- Help the users in exploring additional features.
+- Improve the customer experience, and help reduce costs by lowering the number of support requests.
```

Comment:
**Comment (non-blocking):** Overall, I like your list of benefits. Some more benefits include reducing onboarding time. Because good quickstarts focus on very simple tasks, they give users the feeling that the product is easy to use because they were able to quickly complete their first task with the product.

Reply:
Fixed